packages/socket-mode/src/SocketModeClient.ts

```diff
@@ -573,6 +573,7 @@ export class SocketModeClient extends EventEmitter {
     }
 
     // Define Ack
+    // tslint:disable-next-line:ter-arrow-parens
     const ack = async (response: object): Promise<void> => {
       this.logger.debug('calling ack', event.type);
       await this.send(event.envelope_id, response);
```

Comment:
This was a bit of weird one
![image](https://user-images.githubusercontent.com/10026538/120312173-4b76e000-c2d0-11eb-826a-386ac3836391.png)
`tslint` appears to be confused.
![image](https://user-images.githubusercontent.com/10026538/120313084-74e43b80-c2d1-11eb-9a2c-9993591a2537.png)

I tried to "refactor" the implementation by putting the function signature into a `type` to try and trick `tslint` but it didn't work
```typescript
    // Define Ack
    type ackSig = (response: object) => Promise<void>;
    const ack: ackSig = async (response) => {
      this.logger.debug('calling ack', event.type);
      await this.send(event.envelope_id, response);
    };
```

You can't simply remove the arrow syntax as it is needed for the closure & `this`. I had a little play around but gave up and thought can we instead just ignore it? What do you think about adding an ignore for this @seratch?

```typescript
    // Define Ack
    // tslint:disable-next-line
    const ack = async (response: object): Promise<void> => {
      this.logger.debug('calling ack', event.type);
      await this.send(event.envelope_id, response);
    };
```

Reply:
>can we instead just ignore it? What do you think about adding an ignore for this @seratch?

Thanks for checking this in details. Ignoring the lint warning sounds fine 👍